### ISSUE TITLE
Maven goal name should be 'azure-spring-cloud:deploy'

### ISSUE THREAD

**Issue by ZhengJun003** [microsoft/azure-maven-plugins#824]
### Plugin name and version
```
azure spring cloud maven plugin
0.1.0-SNAPSHOT
```
### Plugin configuration in your `pom.xml`
```
<plugin>
  <groupId>com.microsoft.azure</groupId>
  <artifactId>azure-spring-cloud-maven-plugin</artifactId>
  <version>0.1.0-SNAPSHOT</version>
</plugin>
```

### Actual behavior
![图片](https://user-images.githubusercontent.com/41051257/65224479-d791a580-daf5-11e9-829a-7cec22a854bf.png)

**Comment by Flanker32** [comment]
Fixed in #827

**Comment by ZhengJun003** [comment]
Fixed
Test in develop branch
